docs/product-assembly_migrating-6.4-to-7.x/src/main/asciidoc/main.adoc

```diff
@@ -0,0 +1,78 @@
+[id='assembly_migrating-6.4-to-7.x']
+
+include::product-shared-docs/document-attributes.adoc[]
+
+= Migrating from {PRODUCT_OLD} 6.4 to {PRODUCT} {PRODUCT_VERSION}
+// Context attribute is assembly specific and enables module reuse between assemblies.
```

Comment:
@sterobin - is "assembly specific" a hyphenate?

Reply:
@seanhuck, not like this. Only hyphenated when used as a compound adj. Example: "an assembly-specific context" vs. "is assembly specific". Glad to see you commenting here! :)